script/make_utils/fix_omp_issues_for_intel_mac.sh

```diff
@@ -21,20 +22,47 @@ then
     WHICH_VENV=$(command -v python | sed -e "s@bin/python@@")
     WHICH_PYTHON=$(python -c 'import sys; print(f"python{sys.version_info.major}.{sys.version_info.minor}")')
 
+    # To update the dylib changes (eg with major updates in torch), please uncomment this, and
+    # then recopy to the given sections
+    if [ $DO_REGENERATE -eq 1 ]
```

Comment:
this is the new option to regenerate automatically the dylib fixes with new versions